docs/src/documentation/05-development/04-migration-guides/01-v23.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: v23
+description: How to migrate to Orbit 23.0.0
+redirect_from:
+  - /migration-guides/v23/
+---
+
+# Orbit Migration Guide v23
+
+This migration guide focuses on the process of migrating from Orbit v22 to v23.0, as some breaking changes were introduced.
+With this guide, we aim to walk through all the breaking changes and how they can be addressed, allowing the migration to be smoother and effortlessly.
+
+## Breaking changes
+
+### Title prop for Skeleton component
+
+The `Skeleton` component used to have a default value for the prop `title`. We removed this default value, so now you need to provide it.
```

Comment:
> We removed this default value, so now you need to provide it.

If I understood correctly, this is not a requirement, isn't it?

Reply:
It's not, I rewrote the sentence.